README.md

```diff
@@ -24,40 +24,58 @@ You will require unleash on your class path, pop it in to your pom:
 ```
 
 
-### Create a new Unleash instance
+### Step 2: Create a new Unleash instance
 
-It is easy to get a new instance of Unleash. In your app you typically *just want one instance of Unleash*, and inject that where you need it. You will typically use a dependency injection frameworks such as Spring or Guice to manage this.
+---
 
-To create a new instance of Unleash you need to pass in a config object:
-```java
+**⚠️ Important:** In almost every case, you only want a **single, shared instance of the `Unleash`  class (a *singleton*)** in your application . You would typically use a dependency injection framework (such as Spring or Guice) to inject it where you need it. Having multiple instances of the client in your application could lead to inconsistencies and performance degradation.
+
+To help you detect cases where you configure multiple instances by mistake, the SDK will print an error message if you create multiple instances with the same configuration values. You can also tell Unleash to fail when this happens by setting the constructor parameter `failOnMultipleInstantiations` to `true`.
+
+---
+
+When instantiating an Unleash client, you can choose to do it either **synchronously** or **asynchronously**:
+The SDK will synchronize with the Unleash API on initialization, so it can take a few hundred milliseconds for the client to reach the correct state.
+This is usually not an issue and Unleash will do this in the background as soon as you initialize it.
+However, if it's important that you not continue execution until the SDK has synchronized, then you should use the `synchronousFetchOnInitialisation` option to block the client until it has successfully synchronized with the server.
+
+### Example configurations
+
+**💡 Tip:** Refer to the section on [configuration options](#configuration-options) for a more complete explanation of all the options.
 
+Here's two examples of how you might initialize the Unleash SDK in your applications. The examples use dummy values and are almost identical. The only difference is that the first example is asynchronous, while the second example is synchronous.
+
+**Asynchronous initialization example:**
+
+```java
 UnleashConfig config = UnleashConfig.builder()
-            .appName("java-test")
-            .instanceId("instance x")
-            .unleashAPI("http://unleash.herokuapp.com/api/")
-            .apiKey("API token")
-            .build();
+        .appName("my.java-app")
+        .instanceId("your-instance-1")
+        .unleashAPI("<unleash-api-url>")
+        .customHttpHeader("Authorization", "<client-api-token>")
```

Comment:
Still recommend using `apiKey("<client-api-token")` instead of `customHttpHeader("")`